test/configuration_reader_test.exs

```diff
@@ -47,6 +47,15 @@ defmodule ExsodaTest.ConfigurationReader do
                                                                             value: %{"enabled" => true}},
                                                     %Configuration.Property{name: "fun",
                                                                             value: %{"enabled" => true}}],
-                                       type: "view_categories"}]
+                                       type: "view_categories"},
+                        %Configuration{id: 897,
+                                       name: "View categories",
```

Comment:
Why do these require the `Exsoda` where the first item in this list doesn't? Also maybe I'm misunderstanding but this doesn't test the change you made (to lowercase the id) right?